react-hs/src/React/Flux/Addons/Intl.hs

```diff
@@ -389,7 +389,7 @@ utcTime_ (dayFmt, timeF) t = time_ [property "dateTime" timeRef] $ foreignClass
 -- object and passed in the @value@ property.  The remaining properties can be any properties that
 -- <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DateTimeFormat Intl.DateTimeFormat>
 -- accepts.  For example, you could pass in \"timeZone\" to specify a specific timezone to display.
-formattedDate_ :: Either Day UTCTime -> [PropertyOrHandler eventHandler] -> ReactElementM eventHandler ()
+--formattedDate_ :: Either Day UTCTime -> [PropertyOrHandler eventHandler] -> ReactElementM eventHandler ()
```

Comment:
if this is not correct any more, why not add the new signature?  or remove this one instead of commenting it out?

Reply:
I couldn't get it to compile. Don't know why. Probably just me, but might of course be a GHC issue. 
The correct solution would of course be to fix it instead of remove it, so I wanted to leave it commented if someone else could figure out how why it doesn't compile